src/main/webapp/app/shared/metis/metis-conversation.service.ts

```diff
@@ -30,6 +31,8 @@ export class MetisConversationService implements OnDestroy {
     // Stores the currently selected conversation
     private _activeConversation: ConversationDto | undefined = undefined;
     _activeConversation$: ReplaySubject<ConversationDto | undefined> = new ReplaySubject<ConversationDto | undefined>(1);
+    private _hasUnreadMessages = false;
+    _hasUnreadMessages$: Subject<boolean> = new ReplaySubject<boolean>(1);
```

Comment:
Is there a special reason for the underscore prefix? Otherwise see
https://docs.artemis.cit.tum.de/dev/guidelines/client/?highlight=private#names

Reply:
The reason was to be consistent with other variables within this file.

Reply:
I would agree, that we should not use underscore prefixes

Reply:
@krusche do you suggest to change all the variables with underscore prefix within this file?

Reply:
That should be easy doable by using the refactor > renaming option in IntelliJ, so I don't see a reason for not doing so